src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/BulkScanController.java

```diff
@@ -40,6 +40,9 @@
 @RequiredArgsConstructor
 public class BulkScanController {
```

Comment:
Please use 0.0.13 and then you can refactor the way we define endpoints (using `BulkScanEndpoints` class)